iotdb/iotdb/conf/iotdb-engine.properties

```diff
@@ -25,6 +25,18 @@ rpc_address=0.0.0.0
 
 rpc_port=6667
 
+####################
+### Dynamic Parameter Adapter Configuration
+####################
+
+# Is dynamic parameter adapter enable. It's recommended for users to enable parameter adapter.
+# The adapter can dynamically adjust the following two parameters according to the memory load of the system:
+# 1. tsfile_size_threshold which is introduced below.
+# 2. group_size_in_byte which is introduced in tsfile-format.properties.
```

Comment:
It would be better to add a parameter called memtable_size in iotdb-engine.properties, separating this with group_size_in_byte

Reply:
fixed